src/VirtualClient/VirtualClient.Actions/FIO/FioExecutor.cs

```diff
@@ -840,6 +840,19 @@ protected virtual void CaptureMetrics(
             }
 
             IList<Metric> metrics = parser.Parse();
+            string fioVersion = null;
+
+            var fioVersionMetric = metrics.FirstOrDefault(m => m.Name == "IsFioVersionCaptured");
+            if (fioVersionMetric != null && fioVersionMetric.Metadata.TryGetValue("fio version", out var versionValue))
+            {
+                fioVersion = versionValue?.ToString();
+            }
+
+            if (!string.IsNullOrEmpty(fioVersion))
+            {
+                this.MetadataContract.Add("fio_version", fioVersion, MetadataContractCategory.Dependencies);
```

Comment:
We have a standard way to define the toolset version for the tool associated with the metrics. It is ok to add it to the metadata_dependencies section as well but add it to the LogMetrics call as well to be complete.

![image](https://github.com/user-attachments/assets/a71b70da-7453-4a04-88de-51f5b69b5aaa)